src/bin/cql-stress-cassandra-stress/settings/param/distribution.rs

```diff
@@ -0,0 +1,153 @@
+use std::collections::HashMap;
+
+use anyhow::Result;
+
+use crate::java_generate::distribution::DistributionFactory;
+
+use super::Param;
+
+/// Distribution-specific arguments parser.
+trait DistributionParser: Sync {
```

Comment:
We already have `src/distribution.rs`, can we reuse it? Cassandra-stress and scylla-bench use similar syntax for distributions, so I wrote that code keeping in mind that it might be possible to use it for the c-s frontend.